code/go/0chain.net/blobbercore/writemarker/entity.go

```diff
@@ -19,6 +19,9 @@ type WriteMarker struct {
 	Timestamp              common.Timestamp `gorm:"column:timestamp" json:"timestamp"`
 	ClientID               string           `gorm:"column:client_id" json:"client_id"`
 	Signature              string           `gorm:"column:signature" json:"signature"`
+	Name                   string           `gorm:"column:name" json:"name"`
+	LookupHash             string           `gorm:"column:lookup_hash" json:"lookup_hash"`
+	ContentHash            string           `gorm:"column:content_hash" json:"content_hash"`
```

Comment:
it is unnecessary to save content_hash and name on `write_markers` table to save disk space on postgres. 
we can read them from db by lookup_hash when it is sent to blockchain.